x/ibc/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -54,6 +54,13 @@ func (cs ClientState) VerifyUpgradeAndUpdateState(
 			&ConsensusState{}, upgradedConsState)
 	}
 
+	// counterparty chain must commit the upgraded client with all client-customizable fields zeroed out
+	// at the upgrade path specified by current client
+	// counterparty must also commit to the upgraded consensus state at a sub-path under the upgrade path specified
+	if !ZeroedCustomFields(tmUpgradeClient) {
```

Comment:
I suppose this doesn't matter since we can always zero on behalf of governance but whatever